package.json

```diff
@@ -32,15 +37,14 @@
     "url": "git+https://github.com/contentful/starter-gatsby-blog.git"
   },
   "scripts": {
-    "dev": "gatsby develop",
+    "dev": "npm run develop",
```

Comment:
I wonder why they kept these separate 🤔 I think we can just merge `dev` and `develop`.